### DISCUSSION TITLE
重庆联通 IPTV 单线复用 + 内网融合教程

### DISCUSSION THREAD

**Discussion by 1715173329** [1715173329/blog#15] [Comments]
https://blog.imouto.in/post/iptv/2022/cqcu-iptv-on-openwrt/

**Comment by lalasou** [comment]
大神你好
请问  路由器和光猫之间一根网线，光猫桥接路由器vlan拨号
老版本openwrt交换机
回传光猫扩展lan(把光猫当交换机,光猫lan口就能用了) 
现在能正常上网了
但是这个cpu 这里应该  tag 还是untag
![8CFCA314-D9FE-4122-B6F4-2BD9E4101101](https://user-images.githubusercontent.com/7440313/203997342-6adf5d95-4594-4eee-a032-0082fc52eebd.jpeg)

小白求大佬帮助🙏

**Comment by feheadls** [comment]
重庆联通只有PPPOE拨号才有组播信号，组播信号是在PPPOE虚拨通道中的。

**Comment by fanyunxu** [comment]
请问下  作者使用的什么版本的 openwrt 固件  可以分享下吗

**Comment by fanyunxu** [comment]
重庆九龙坡 联通 成功播放IPTV   九龙坡的tag是3186

**Comment by namtsorye** [comment]
绑定了机顶盒MAC 然后拨号 如何破

**Comment by xkaers** [comment]
> 绑定了机顶盒MAC 然后拨号 如何破

网络-> 设备 中新增一个网络设备然后把机顶盒MAC设置进去就可以了吧. 可以试一下

**Comment by hbmask** [comment]
这个可以开个一群了

**Comment by Ciroygo** [comment]
成功使用，但是比较奇怪的是我自己抓的只有120多个台，怎么作者抓的有140多个台

**Comment by hbmask** [comment]
您的来信我已经收到，我会尽快回复你。

**Comment by qqoicq** [comment]
大佬，没有开通IPTV的重庆联通 单播是不是访问不了了。

**Comment by Honglong22** [comment]
大佬！有没有移动的教程呀？

**Comment by hbmask** [comment]
您的来信我已经收到，我会尽快回复你。

**Comment by 1715173329** [comment]
@hbmask 请停止在本 issue 下面发送无意义的回复，否则你将被 block。

**Comment by Honglong22** [comment]
大佬，中兴路由器没有刷机这种不行是不是

**Comment by Hightree007** [comment]
我也是重庆联通宽带，怎么播放不了单播源？

**Comment by hbmask** [comment]
您的来信我已经收到，我会尽快回复你。

**Comment by JeremyYuri** [comment]
天灵大佬，九龙坡联通，wan口可以获取到ipv6地址，但无法访问ipv6外网，有解决办法吗？
---------------------------------------------------------------------------------------------------------------
解决办法：把udpxy换成msd_lite，后者支持ipv6

**Comment by zhao807699361** [comment]
爱快可以设置来用吗？我设置了不能用，iptv拨上了有ip了，但是看不了
